### DISCUSSION TITLE
Bug or feature

### DISCUSSION THREAD

**Discussion by DevVigilante** [sketchide/SketchIDE#38] [General]
It is a bug or feature??

![IMG_20240325_163638](https://github.com/androidbulb/SketchIDE/assets/96680248/bf8f3948-ba49-4480-8ce9-9ca3e787e1da)

Also the project is only running in Android Studio and IntelliJ IDEA so please setup GitHub action so we can download apk file from GitHub for testing purpose as we need pc/laptop to install apk😔.

**Comment by CyberGlitch01** [comment]
@androidbulb

**Reply by DevVigilante** [reply]
I think this app will no longer built

**Reply by CyberGlitch01** [reply]
I think so

**Comment by itsvks19** [comment]
> It is a bug or feature??
> 
> ![IMG_20240325_163638](https://private-user-images.githubusercontent.com/96680248/316479460-bf8f3948-ba49-4480-8ce9-9ca3e787e1da.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3NjY1ODEsIm5iZiI6MTcxMTc2NjI4MSwicGF0aCI6Ii85NjY4MDI0OC8zMTY0Nzk0NjAtYmY4ZjM5NDgtYmE0OS00NDgwLThjZTktOWNhM2U3ODdlMWRhLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzMwVDAyMzgwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI3NWI3NGEwNDg0MGJkYjliMWEzMDJlMjVkY2Y0NjVkYWE1NWUxN2ZiMzkxYTZlMGUzMGVhNmY3NzkzODQxNjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jqhoTY2qF2EFJZAAjGBeDgSU9wlBNoN1NZqt7cIoEhM)
> 
> Also the project is only running in Android Studio and IntelliJ IDEA so please setup GitHub action so we can download apk file from GitHub for testing purpose as we need pc/laptop to install apk😔.

Add a ViewGroup to add more widgets like LinearLayout.

**Reply by DevVigilante** [reply]
I thought it has a parent by default.
